pkg/config/config.go

```diff
@@ -84,6 +84,11 @@ func SelectProfile(name string) bool {
 	return result
 }
 
+func SelectDefaultProfile() bool {
+	log.Warnf("{warning:Using default profile}")
```

Comment:
Minor nit - this and the warning on line 325 have no space after `warning:`

Reply:
@ClassicStocksy Warning doesn't show in the message, it's a formatting directive in the logger functions. This will highlight the text in yellow in a terminal session. The message is displayed as

`[ Warn] Using default profile`